sdk/core/azure-core-amqp/vcpkg/vcpkg.json

```diff
@@ -20,7 +20,7 @@
     {
       "name": "azure-core-cpp",
       "default-features": false,
-      "version>=": "1.10.0"
```

Comment:
We can't put beta versions here, https://github.com/Azure/azure-sdk-for-cpp/pull/4881, remember, in August, I think, when I was OOF, you and Larry ran into this, if I am not mistaking?
For vcpkg, even if the version is specified as "version-semver", if the version is not present in the vcpkg official repo (which we don't do for post-GA libs), it triggers an error.

Reply:
That's true, and because we haven't GA'd AMQP yet, when we beta it and it goes in the main registry, it won't have Core.

Luckily, AMQP and OpenTelemetry don't depend on this upcoming version of Core beta. So, I will remove that. Good catch!

Reply:
To confirm: The packages that are shipping a post-GA beta (core, identity, etc.) should be fine, correct, if we put a core beta version as a dependency that will exist in our beta vcpkg registry?

Reply:
Yes, if no package with Core Beta dependency will be pushed to vcpkg official, it will be fine.